sw/device/lib/testing/flash_ctrl_testutils.h

```diff
@@ -255,6 +255,15 @@ status_t flash_ctrl_testutils_backdoor_init(
  * a workaround to invalidate the cache and force it to update. Before using
  * this function `flash_ctrl_testutils_backdoor_init` should be called.
  *
+ * Note:
+ * Given typical flash write latency (a few 10s of us),
+ * this api can causes following misbehavior with real flash model.
+ *
+ * While flash write with all 1's in progress, backdoor write from
+ * sv finished. When that happens, this function can complete without
+ * detectiing proper backdoor update value. So it is highly discouraged to use
+ * this api without opensource flash model.
```

Comment:
Can you open an M2.5 backlog issue and link it here? 
This sounds like something we should fix at some point.

Also mini nit: `detectiing`

Reply:
Updated.
issue #19122 is created too.